### PR TITLE
Python3 compatibility

### PR DESCRIPTION
**Pull request by tswfi**
on top of #1 fixes the current test suite to run on python3. No other checks have been made.

### CODE REVIEW

File: epages/auth.py [hunk 1]
```diff
@@ -8,14 +8,22 @@
 
 
 def calculate_signature(code, access_token_url, client_secret):
-    message = '%s:%s' % (code, access_token_url)
-    digest = hmac.new(client_secret, msg=message, digestmod=hashlib.sha256).digest()
-    return base64.b64encode(digest).decode()
+    message = b'%s:%s' % (code, access_token_url)
```
Comment (ooz) [inline epages/auth.py]: I'd rather pass (unicode) strings as parameters and encode to bytestrings in the function to have an easier-to-use interface

File: tests/test_auth.py [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
 from tests.context import epages
 
-CODE = 'rp6YlWEcSxS81OTmdksLOaqv7TRbpzSV'
-ACCESS_TOKEN_URL = 'https://devshop.epages.com/rs/shops/lingering-hill-2037/token'
-CLIENT_SECRET = 'pM8vwvhYTNMErQsNSJZ0wr91ucOmL5JI'
-SIGNATURE = 'g1L73Mv6GnUE8vfWHmxKQmG9KuLiwwI9LMR50w+SXTo='
+CODE = b'rp6YlWEcSxS81OTmdksLOaqv7TRbpzSV'
+ACCESS_TOKEN_URL = b'https://devshop.epages.com/rs/shops/lingering-hill-2037/token'
+CLIENT_SECRET = b'pM8vwvhYTNMErQsNSJZ0wr91ucOmL5JI'
+SIGNATURE = b'g1L73Mv6GnUE8vfWHmxKQmG9KuLiwwI9LMR50w+SXTo='
```
Comment (ooz) [inline tests/test_auth.py]: See comment above, please value the method contract and encode strings to bytestrings in the method

**Review by ooz — CHANGES_REQUESTED** [review]

**Comment by tswfi** [comment]
Like that? I don't usually use 2 so I'm not that familiar on how the string handling works there :)

**Comment by ooz** [comment]
Looks good to me.

Problem now though: If I merge this it breaks Python 2 compatibility, because
```
except epages.RESTError as error:
```
is Python 3 only.

A solution to this would probably be some compatibility library (2to3, six?)

**Comment by ooz** [comment]
You are correct!

Wierd, I'm certain, I tried this some time ago and it did not work.

**Review by ooz — APPROVED** [review]

**Pull request merged**